restructuredtext_lint/test/test.py

```diff
@@ -1,11 +1,13 @@
 import os
+from subprocess import check_output, CalledProcessError
 from unittest import TestCase
 
 import yaml
 
 import restructuredtext_lint
 
-__dir__ = os.path.dirname(os.path.abspath(__file__))
+
+rst_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'test_files')
```

Comment:
If we are going to support `os.path.join`, then we should clean up the slashes in the other paths as well:

```python
__dir__ = os.path.dirname(os.path.abspath(__file__))
# ...
filepath = os.path.join(__dir__, 'test_files', 'invalid.rst')
```